src/reduxFiles/reducers/playlists/playlistsSlice.js

```diff
@@ -0,0 +1,263 @@
+    );
+  }
+};
+
+const createNewPlaylistHandler = async (
+  { playlistDetailsGiven, tokenProvided },
+  { rejectWithValue }
+) => {
+  try {
+    const {
+      data: { playlists: playlistsReceivedFromResponse },
+    } = await createNewPlaylistService(playlistDetailsGiven, tokenProvided);
+    return playlistsReceivedFromResponse;
+  } catch (createNewPlaylistError) {
+    console.error("AN ERROR OCCURRED WHILE CREATING A PLAYLIST THROUGH REDUX");
+    return rejectWithValue(
+      "Error occured in creating this playlist. Please try again"
+    );
+  }
+};
+
+const deleteExistingPlaylistHandler = async (
+  { playlistDetailsGiven, tokenProvided },
+  { rejectWithValue }
+) => {
+  try {
+    const {
+      data: { playlists: playlistsReceivedFromResponse },
+    } = await deleteExistingPlaylistService(
+      playlistDetailsGiven,
+      tokenProvided
+    );
+    return playlistsReceivedFromResponse;
+  } catch (createNewPlaylistError) {
+    console.error("AN ERROR OCCURRED WHILE DELETING A PLAYLIST THROUGH REDUX");
+    return rejectWithValue(
+      "Error occured in deleting this playlist. Please try again"
+    );
+  }
+};
+
+const receiveSinglePlaylistDetailsHandler = async (
+  { playlistDetailsGiven, tokenProvided },
+  { rejectWithValue }
+) => {
+  try {
+    const {
+      data: { playlist: playlistDataReceivedFromResponse },
+    } = await receiveSinglePlaylistDetailsService(
+      playlistDetailsGiven,
+      tokenProvided
+    );
+    return playlistDataReceivedFromResponse;
+  } catch (receiveSinglePlaylistDetailsError) {
+    console.error(
+      "AN ERROR OCCURRED WHILE FETCHING A SINGLE PLAYLIST DETAILS THROUGH REDUX"
+    );
+    return rejectWithValue(
+      "Error occured in fetching this playlist details. Please try again"
+    );
+  }
+};
+
+const addNewVideoToPlaylistHandler = async (
+  { playlistDetailsGiven, videoToBeAdded, tokenProvided },
+  { rejectWithValue }
+) => {
+  try {
+    const {
+      data: { playlist: playlistDataReceivedFromResponse },
+    } = await addNewVideoToPlaylistService(
+      playlistDetailsGiven,
+      videoToBeAdded,
+      tokenProvided
+    );
+    return playlistDataReceivedFromResponse;
+  } catch (addNewVideoToPlaylistError) {
+    console.error(
+      "AN ERROR OCCURED WHILE ADDING THIS VIDEO TO THIS PLAYLIST THROUGH REDUX"
+    );
+    return rejectWithValue(
+      "Error occured while adding this video to a playlist. Please try again"
+    );
+  }
+};
+
+const deleteExistingVideoFromPlaylistHandler = async (
+  { playlistDetailsGiven, videoToBeDeleted, tokenProvided },
+  { rejectWithValue }
+) => {
+  try {
+    const {
+      data: { playlist: playlistDataReceivedFromResponse },
+    } = await deleteExistingVideoFromPlaylistService(
+      playlistDetailsGiven,
+      videoToBeDeleted,
+      tokenProvided
+    );
+    return playlistDataReceivedFromResponse;
+  } catch (deleteExistingVideoFromPlaylistError) {
+    console.error(
```

Comment:
You can remove the console log from all the places

Reply:
Have kept it for debugging purposes here as it would be helpful to understand where the error came up from.